cmd/boost/direct_deal.go

```diff
@@ -285,3 +288,291 @@ func printAllocation(allocations map[verifreg.AllocationId]verifreg.Allocation,
 		return tw.Flush(os.Stdout)
 	}
 }
+
+var clientExtendDealCmd = &cli.Command{
+	Name:  "extend-claim",
+	Usage: "extend claim expiration (TermMax)",
+	Flags: []cli.Flag{
+		&cli.Int64Flag{
+			Name:    "term-max",
+			Usage:   "The maximum period for which a provider can earn quality-adjusted power for the piece (epochs). Default is 5 years.",
+			Aliases: []string{"tmax"},
+			Value:   verifreg13types.MaximumVerifiedAllocationTerm,
+		},
+		&cli.StringFlag{
+			Name:  "wallet",
+			Usage: "the wallet address that will used to send the message",
+		},
+		&cli.BoolFlag{
+			Name:  "all",
+			Usage: "automatically extend TermMax of all claims for specified miner[s] to --term-max (default: 5 years from claim start epoch)",
+		},
+		&cli.StringSliceFlag{
+			Name:    "miner",
+			Usage:   "storage provider address[es]",
+			Aliases: []string{"m", "provider", "p"},
+		},
+		&cli.BoolFlag{
+			Name:    "assume-yes",
+			Usage:   "automatic yes to prompts; assume 'yes' as answer to all prompts and run non-interactively",
+			Aliases: []string{"y", "yes"},
+		},
+		&cli.IntFlag{
+			Name:  "confidence",
+			Usage: "number of block confirmations to wait for",
+			Value: int(build.MessageConfidence),
```

Comment:
` --confidence value                                                                                           number of block confirmations to wait for (default: 5)`

Reply:
--help will convert the value and show the default to 5 automatically. In other places, it had to write the default because --help would show the value in epochs which is not that useful.